### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by forseyra** [basildane/WakeOnLAN#8]
Installed on Aldi Windows Tablet running Windows 8.1

When run:

An unexpected exception occurred.

Object reference not set to an instance of an object.

Stack Trace
at WakeOnLan.My.MyApplication.ConfigureCulture()
   at WakeOnLan.My.MyApplication.MyApplication_Startup(Object sender, StartupEventArgs e)
   at Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.OnStartup(StartupEventArgs eventArgs)
   at Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.DoApplicationModel()

**Comment by basildane** [comment]
Thanks for reporting it.
It seems to be having a problem setting the language / culture.

Could I ask, what is your native language on that computer?

And did you run WOL at all and change the language, or is this the first run?

**Comment by basildane** [comment]
And also, which version of the software do you have?

**Comment by basildane** [comment]
I have managed recreated the problem here.  I am looking for the cause now.

**Comment by basildane** [comment]
[WakeOnLAN_2.11.6.16.zip](https://github.com/basildane/WakeOnLAN/files/213200/WakeOnLAN_2.11.6.16.zip)
The bug was deep in the installer.  This should fix it.  It was a permissions problem with non-admin users installing the software.  I would appreciate it if you let me know if it works for you.  I have attached the beta installer.

**Comment by forseyra** [comment]
Thanks for the prompt response.

1. Version that was available for download on Sunday.

2. Language here in Australia is English.

3. It was installed using a non-administrative account. I had to supply the password for an admin account to install.

4. You need to update the screen shot in the Online help for WOL over Internet page “Properties – Scratchy” to match your latest release.

5. I did try running WakeOnLAN using Run as administrator yesterday and it ran OK.

5. I will visit the site later today and test your installer fix.

**Comment by forseyra** [comment]
1. Installed the beta installer version using a non-privileged account  and WakeOnLAN started OK after the installation. As expected, it still asked for the credentials of an administrative account.
2. I also tested the installation by right clicking on the install executable and selecting Run as administrator. Again, WakeOnLAN ran OK under a non-privileged account.
3. Ran Listener and the following appeared:

4. Ran Netstat –ab in adminsrative command window and confirmed that nothing is using UDP 0.0.0.0:9
Same problem evident on my home Windows 10 PC.
5. Tried to have a look at what was happening on the LAN, but found that Microsoft Network Monitor no longer runs under Windows 10 Version 1511.
Installed Microsoft Message Analyzer but it didn’t capture any network packets on my Toshiba laptop. Similar problems on home Windows 10 Desktop. It does run on my old Lenovo T61 running Windows 10!
I’ll do some more testing tomorrow and send you the results.
Roger Forsey
